### PR TITLE
CO-3195 / CO-3196 Add gift payment slips to support replies

### PR DESCRIPTION
**Pull request by SylvainLosey**

### CODE REVIEW

File: crm_request/models/mail_compose_message.py [hunk 1]
```diff
@@ -38,14 +40,61 @@ def send_mail(self, auto_commit=False):
...
         """
-        Append the quote of previous e-mail to the body of the message.
+        - Append the quote of previous e-mail to the body of the message.
+        - Add gift payment slips for "Gift/Payment slip" and "Christmas Gift Fund"
         """
         result = super().onchange_template_id(
             template_id, composition_mode, model, res_id
         )
+
+        # Add reply qyote
```
Comment (Theyiot) [inline crm_request/models/mail_compose_message.py]: Little typo in the comment.
```suggestion
        # Add reply quote
```

**Review by Theyiot — CHANGES_REQUESTED** [review]
Except for the typo in the comment, the rest looks fine to me.

**Comment by ecino** [comment]
@Theyiot Can you move these changes into crm_switzerland (as explained in JIRA issue that I reopened and assigned to you).

**Comment by Theyiot** [comment]
> @Theyiot Can you move these changes into crm_switzerland (as explained in JIRA issue that I reopened and assigned to you).

I'm working on it.

**Pull request closed without merging**